aw-datastore/src/datastore.rs

```diff
@@ -581,4 +598,49 @@ impl DatastoreInstance {
 
         return Ok(count);
     }
+
+    pub fn create_value(&self, conn: &Connection, key: &str, data: &str)
+        -> Result<(), DatastoreError> {
+
+        let mut stmt = match conn.prepare("
```

Comment:
Maybe add an limit on how long keys can be? 100 chars?
SQLite TEXT does not have a limit, but we should probably add one anyway.

The values can be unlimited.

Reply:
I'm thinking of adding a check to the api endpoints like this ```if key.len() > 128 { return Err(Status::BadRequest) }``` (128 is a nice and familiar number for max size imo. 100 is fine as well.)

I wouldn't think the internal implementation needs a check if we don't allow too long inputs, but could add that if it's necessary?

Reply:
Sounds fine!

Reply:
Implemented and fixed in fdc4554